### DISCUSSION TITLE
Apply a local patch to a package

### DISCUSSION THREAD

**Discussion by kanwarplaha** [mocaccinoOS/mocaccino#89] [Q&A] [answered]
Can we apply a local patch to an upstream package, such as adding a diff under `/etc/portage/patches/....`?

Very new to MocaccinoOS so any help on how to do this would be great. Thanks.

**Comment by joostruis** [comment]
Can you please explain what you are trying to achieve? 
When building things we can utilize that yes.

In the commit below you can see how I done that.
https://github.com/mocaccinoOS/desktop/commit/eb992d889be2e5ba32971b89360970386b38a2b8

**Reply by kanwarplaha** [reply]
I am trying to patch kwin as per https://gist.github.com/sayap/f343bff7700d45b555aea7e581097314. The author of that did it on gentoo so I wanted to know how I can achieve the same on MocaccinoOS :)

Thanks.

**Comment by joostruis** [comment, marked as answer]
You can achieve this only if you create your own local repository using Luet.
MOS does not offer support for Portage.
